### PR TITLE
feat(theme): Add `searchBox` widget to default theme

### PR DESCRIPTION
**Pull request by pixelastic**
- Set styling of the `poweredBy` as part of the `search-box` widget
- Move appropriate styling to `default.css`
- Update readme
- Use image replacement technique and base64 in the theme file
- Placeholder in theme file for other widgets to come

BREAKING CHANGE: "Powered by" styles are now
`ais-search-box--powered-by` and `ais-search-box--powered-by-link`.

### CODE REVIEW

**Comment by redox** [comment]
LGTM :+1:

File: example/default.css [hunk 1]
```diff
@@ -0,0 +1 @@
+../themes/default/default.css
```
Comment (vvo) [inline example/default.css]: what does this means?

Comment (pixelastic) [inline example/default.css]: It's a symlink, this is how Github displays them I guess.

In a next iteration we'll really add the file from the build.

Comment (vvo) [inline example/default.css]: > In a next iteration we'll really add the file from the build.

Ah yes, we will need to add some css build to the dev task to do so. To be able to modify css live.

Dunno what webpack can bring here.

**Pull request merged**